makefile

```diff
@@ -0,0 +1,5 @@
+all:
+	@ cd server \
+		&& govendor sync \
```

Comment:
you are right, but not is this case

if you are building binary file for production, you should have 2 container. one with all dependencies will build binary, and other one, `from alpline` will just run it

here, you run tests, so all deps must be inside docker, otherwise, why we need it? to run govendor, we need `go` and `govendor` on host machine. this dockerfile is like makefile